### PR TITLE
Update libs with `Statistics` endpoints

### PR DESCRIPTION
**Pull request by jaymell**
Add support for two recently added `Statistics` endpoints:
* Aggregate App Stats 
* Aggregate Event Types

### CODE REVIEW

**Comment by jaymell** [comment]
Note that `Aggregate Event Types` is technically still hidden in our API documentation. Should we fix that?

File: javascript/src/index.ts [hunk 1]
```diff
@@ -875,3 +881,25 @@ export class Webhook {
...
+
+  public constructor(config: Configuration) {
+    this.api = new StatisticsApi(config);
+  }
+
+  public aggregateEventTypes(): Promise<AggregateEventTypesOut> {
+    return this.api.aggregateEventTypes({});
+  }
+
+  public calculateAggregateAppStats(
```
Comment (tasn) [inline javascript/src/index.ts:896]:
```suggestion
  public aggregateApplicationMessages(
```
Stats and calculate are redundant. Need to make sure the event type name is also correct for this, but this should be the name of the function all around.

Comment (jaymell) [reply javascript/src/index.ts:896]: Shall I change the name in our API docs then as well?

Comment (tasn) [reply javascript/src/index.ts:896]: Yup, but that's a one-line change, no?

File: go/statistics.go [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+func (s *Statistics) CalculateAggregateAppStats(ctx context.Context, appUsageStatsIn *AppUsageStatsIn, options *PostOptions) (*AppUsageStatsOut, error) {
+	req := s.api.StatisticsApi.CalculateAggregateAppStats(ctx)
+	if appUsageStatsIn != nil {
+		req = req.AppUsageStatsIn(*appUsageStatsIn)
+	}
+	if options != nil {
+		if options.IdempotencyKey != nil {
+			req = req.IdempotencyKey(*options.IdempotencyKey)
+		}
+	}
```
Comment (svix-onelson) [inline go/statistics.go:28]: The double nil check here always bites me. Good job.

File: go/statistics.go [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+	api *openapi.APIClient
+}
+
+type (
+	AppUsageStatsIn        = openapi.AppUsageStatsIn
+	AppUsageStatsOut       = openapi.AppUsageStatsOut
+	AggregateEventTypesOut = openapi.AggregateEventTypesOut
+)
+
+func (s *Statistics) CalculateAggregateAppStats(ctx context.Context, appUsageStatsIn *AppUsageStatsIn, options *PostOptions) (*AppUsageStatsOut, error) {
```
Comment (svix-onelson) [inline go/statistics.go]:
```suggestion
func (s *Statistics) AggregateAppStats(ctx context.Context, appUsageStatsIn *AppUsageStatsIn, options *PostOptions) (*AppUsageStatsOut, error) {
```

Should these drop the "calculate" prefix too? Maybe we only did it for event types? I think I see the same in Java and probably others further down in the diff.

**Review by svix-onelson — COMMENTED** [review]
I see Tom made a similar comment to mine, so maybe the updates are still pending.

File: go/statistics.go (hunk 3 above)
Comment (jaymell) [reply go/statistics.go]: I didn't squash the commits. Is it possible you're looking at changes from the first commit only?

**Review by tasn — APPROVED** [review]

**Review by tasn — APPROVED** [review]

**Pull request merged**